fmriprep_rodents/workflows/bold/resampling.py

```diff
@@ -265,8 +265,8 @@ def init_bold_std_trans_wf(
         described outputs.
 
     """
+    from ...patch.workflows.func import init_bold_reference_wf
```

Comment:
local imports should go after global imports (i.e., those starting with dots to the end)